app/package.json

```diff
@@ -13,6 +13,6 @@
     "drivelist": "11.0.0",
     "keytar": "7.9.0",
     "@ronomon/direct-io": "3.0.1",
-    "usb": "2.5.2"
+    "usb": "2.4.3"
```

Comment:
Just want to verify that this is an intentional downgrade? It's not obviously attached to the device model number adjustment/additions in a clear way.

Reply:
The most recent `node-usb` update broke things - I notified them (https://github.com/node-usb/node-usb/pull/549) and they fixed it, but they haven’t pushed a new release yet.